mttl/models/containers/selectors/arrow_selector.py

```diff
@@ -0,0 +1,55 @@
+from dataclasses import dataclass
+from typing import Dict, Union
+
+import torch
+
+from mttl.models.containers.selectors import (
+    PerTokenSelector,
+    PerTokenSelectorConfig,
+    register_multi_expert_selector,
+)
+from mttl.models.library.expert_library import ExpertLibrary
+from mttl.models.library.library_transforms import ArrowConfig, ArrowTransform
+
+
+def get_arrow_embeddings(
```

Comment:
I would name `compute_arrow_embeddings` as we are not actually getting the embeddings (only their save name)

Reply:
fixed